hack/build-release.sh

```diff
@@ -9,6 +9,6 @@ source $(dirname "$0")/version-util.sh
 
 ytt -f config/ -f config-release -v kapp_controller_version="$(get_kappctrl_ver)" | kbld -f- > ./tmp/release.yml
 
-shasum -a 256 ./tmp/release*.yml
+shasum -a 256 ./tmp/release*.yml > ./tmp/checksums.txt
```

Comment:
can we output and redirect?

Reply:
So something like `shasum -a 256 ./tmp/release*.yml | tee ./tmp/checksums.txt`?

Reply:
Let me know if you meant something different here, but tested on fork and ran successfully including output of shasum in logs.